src/Compilers/CSharp/Test/Emit/PDB/PDBConstantTests.cs

```diff
@@ -326,6 +326,109 @@ void M()
 </symbols>", format: DebugInformationFormat.PortablePdb);
         }
 
+        [Fact, WorkItem(178988, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/178988")]
+        public void TestStringWithNulCharacter_MaxSupportedLength()
+        {
+            const int length = 2031;
```

Comment:
It's defined in C++ in the implementation of diasymreader. The limit is undocumented.